osu.Game/Screens/Multi/Multiplayer.cs

```diff
@@ -51,7 +51,7 @@ public class Multiplayer : OsuScreen, IOnlineComponent
         [Cached]
         private readonly Bindable<FilterCriteria> currentFilter = new Bindable<FilterCriteria>(new FilterCriteria());
 
-        [Resolved]
+        [Resolved(CanBeNull = true)]
```

Comment:
mind explaining these `MusicController` changes? they don't seem to have much to do with the actual functionality added in this PR.

Reply:
The `MultiScreen` test scene would crash immediately after selecting it since `MusicController` is not available, and again during navigation in `LoungeSubScreen`